libbpf-tools/Makefile

```diff
@@ -43,6 +43,7 @@ APPS = \
 	numamove \
 	offcputime \
 	opensnoop \
+	oomkill \
```

Comment:
Please keep this list sorted and add an entry to `.gitignore`.

Reply:
done

Reply:
Fix it in new commit 89ce26bd6ca2c8a689042f76ea3e4340d61536a8